src/bci_build/package/kubectl.py

```diff
@@ -0,0 +1,24 @@
+"""Kubectl Client BCI container"""
+
+from bci_build.os_version import OsVersion
+from bci_build.package import ApplicationStackContainer
+from bci_build.package.versions import get_pkg_version
+
+KUBECTL_CONTAINERS = []
+KUBECTL_SUPPORTED_OS_VERSIONS = [
+    OsVersion.TUMBLEWEED,
+]
+
+for os_version in KUBECTL_SUPPORTED_OS_VERSIONS:
+    KUBECTL_CONTAINERS.append(
```

Comment:
coding style nit: in most other containers we use the list comprehension style, e.g. 

```python

KUBECTL_CONTAINERS = [
  ApplicationStackContainer(....)
  for os_version in (OsVersion.TUMBLEWEED,)
]
```

instead of for/append.